src/components/movieForm/movieForm.js

```diff
@@ -89,55 +126,71 @@ export const MovieForm = ({ movie, onSubmit }) => {
               className="form-input form-placeholder"
               type="text"
               id="movieUrl"
-              name="movieUrl"
+              {...register('movieUrl', {
+                pattern: { value: /^(http|https):\/\/[^ "]+$/, message: 'Not a valid URL' },
+                required: 'Image URL is required'
+              })}
```

Comment:
Write this code in a separate method and call it over here.

Reply:
Created separate method.